spark/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  Copyright (c) 2023 Adevinta
+  
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to deal
+  in the Software without restriction, including without limitation the rights
+  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+  
+  The above copyright notice and this permission notice shall be included in all
+  copies or substantial portions of the Software.
+  
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+  SOFTWARE.
+  -->
+<resources xmlns:tools="http://schemas.android.com/tools" tools:locale="de">
+    <string name="spark_a11y_modal_fullscreen_close">Fenster schließen</string>
+
+    <string name="spark_back_arrow_content_description">Zurück</string>
+
+    <!--Rating-->
+    <plurals name="spark_rating_with_comments_a11y">
+        <item quantity="one">%1$.1f Durchschnittsbewertung aus %2$d Bewertung</item>
+        <item quantity="other">%1$.1f Durchschnittsbewertung aus %2$d Bewertungen</item>
+    </plurals>
+    <plurals name="spark_rating_with_comments_count_label">
+        <item quantity="one">%d Bewertung</item>
+        <item quantity="other">%d Bewertungen</item>
+    </plurals>
+    <string name="spark_rating_a11y">%1$.1f Durchschnittsbewertung</string>
+
+    <string name="spark_user_avatar_content_description">Profilfoto</string>
+
+    <!--Badge-->
+    <string name="spark_exceed_max_badge_number_a11y">Mehr als %1$d neue Mitteilungen</string>
+    <string name="spark_badge_numberless_a11y">Neue Mitteilung</string>
+    <plurals name="spark_badge_a11y">
+        <item quantity="one">%d neue Mitteilung</item>
+        <item quantity="other">%d neue Mitteilungen</item>
+    </plurals>
+
+    <!--TextField-->
+    <string name="spark_textfield_content_description">Pflichtfeld</string>
+    <string name="spark_textfield_delete_a11y">Löschen</string>
```

Comment:
It would be great if we could replace these whe proper regions tags:
(this could be applied on other locales as well)

```suggestion
    <!--region Rating-->
    <plurals name="spark_rating_with_comments_a11y">
        <item quantity="one">%1$.1f Durchschnittsbewertung aus %2$d Bewertung</item>
        <item quantity="other">%1$.1f Durchschnittsbewertung aus %2$d Bewertungen</item>
    </plurals>
    <plurals name="spark_rating_with_comments_count_label">
        <item quantity="one">%d Bewertung</item>
        <item quantity="other">%d Bewertungen</item>
    </plurals>
    <string name="spark_rating_a11y">%1$.1f Durchschnittsbewertung</string>

    <string name="spark_user_avatar_content_description">Profilfoto</string>
    <!--endregion-->

    <!--region Badge-->
    <string name="spark_exceed_max_badge_number_a11y">Mehr als %1$d neue Mitteilungen</string>
    <string name="spark_badge_numberless_a11y">Neue Mitteilung</string>
    <plurals name="spark_badge_a11y">
        <item quantity="one">%d neue Mitteilung</item>
        <item quantity="other">%d neue Mitteilungen</item>
    </plurals>
    <!--endregion-->

    <!--region TextField-->
    <string name="spark_textfield_content_description">Pflichtfeld</string>
    <string name="spark_textfield_delete_a11y">Löschen</string>
    <!--endregion-->
```

Reply:
A good point I'll add them 👍